.github/workflows/run-tests.yml

```diff
@@ -55,6 +55,10 @@ jobs:
     if: contains(github.event.pull_request.labels.*.name, 'run-api-tests')
     steps:
     - uses: actions/checkout@v2
+    - name: Set up JDK 11
```

Comment:
I don't think this will help. War is built inside docker container. As agreed on the backend channel, code should compile on jdk8, unless the team agrees to explicitly require jdk9+

Reply:
I think going forward into 2.36 it makes sense to shift requirement to JDK 11.